pages/add-clinic.vue

```diff
@@ -132,10 +135,10 @@ export default {
             .firestore()
             .collection("pending")
             .add(clinic)
-            .then(() => console.log("added to db", this.clinic))
-            .then(this.$router.push("/"));
+            .then(() => this.$refs.successDialog.open());
```

Comment:
instead of `then()` let's use the `async/await` pattern. 

slack me if you need help and I can pair with you :)